src/shared-components/verificationMessages/index.tsx

```diff
@@ -38,28 +38,29 @@ export const VerificationMessages = ({
   type = 'error',
 }: VerificationMessagesProps) => {
   const messageKeys = Object.keys(messages);
-
-  if (messageKeys.length === 0) {
-    return null;
-  }
+  const showMessages = messageKeys.length > 0;
 
   return (
     <TransitionGroup component={centered ? CenteredMessageList : MessageList}>
-      {messageKeys
-        .filter((key) => {
-          const message = messages[key];
-          if (!Array.isArray(message)) {
-            return true;
-          }
-          return message.length > 0;
-        })
-        .map((key) => (
-          <HelperTransition key={key}>
-            <MessageItem type={type}>
-              {formatMessage(messages[key])}
-            </MessageItem>
-          </HelperTransition>
-        ))}
+      {showMessages ? (
+        messageKeys
+          .filter((key) => {
+            const message = messages[key];
+            if (!Array.isArray(message)) {
+              return true;
+            }
+            return message.length > 0;
+          })
+          .map((key) => (
+            <HelperTransition key={key}>
+              <MessageItem type={type}>
+                {formatMessage(messages[key])}
+              </MessageItem>
+            </HelperTransition>
+          ))
+      ) : (
+        <li />
```

Comment:
Can we return `null` here instead of an empty list item? It looks like we were returning `false` before, so I imagine it should work.

Reply:
tried false and null but TS yells is not compatible with children type which is Element[]

Reply:
tested and it does prints the ul with one empty li and height of 0. I guess is safe, otherwise we could jsut add the ts-ignore and return false like before

Reply:
What about `<React.Fragment />`? I just pulled that in without errors.

Reply:
yeah I forgot.  Fragment looks better I compared to Prod login screen and they both prints empty ul,.
 Changed, updated snapshot and merging soon